rollup.config.js

```diff
@@ -37,6 +38,9 @@ const createConfig = cliArgs => {
       sourcemap: true,
     },
     plugins: [
+      replace({
+        'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
```

Comment:
Hmm we already do this in webpack, I'm not sure if it makes sense to do it here as well.

Reply:
Some others however might not use our webpack config and will not do this or forget to. I think this is a decent safety net?

Reply:
@emmenko do we apply it to node_modules?

Reply:
I'll check the built playground code to see if it achieves the same thing...

Reply:
I'm mostly concerned to "turn off" things in the libraries when they are used in development as well. So it should be up to the application bundler to turn off those things.

For example, if we do this, the logger won't work anymore when we develop apps locally
https://github.com/commercetools/merchant-center-application-kit/blob/dc3795c73216b447e231c5f3748b4b9bf9b0e867/packages/sdk/src/utils/logger.js#L9

Reply:
Yeah I think you are right 👍